pkg/cloudprovider/providers/gce/gce_loadbalancer.go

```diff
@@ -563,6 +592,13 @@ func (gce *GCECloud) targetPoolURL(name, region string) string {
 	return fmt.Sprintf("https://www.googleapis.com/compute/v1/projects/%s/regions/%s/targetPools/%s", gce.projectID, region, name)
 }
 
+func (gce *GCECloud) getLoadBalancerName(service *v1.Service) string {
+        //GCE requires that the name of a load balancer starts with a lower case letter.
```

Comment:
We should do something like:  service.Namespace[0:13]-service.Name[0:13]-service.UID